packages/grafana-prometheus/src/components/AnnotationQueryEditor.tsx

```diff
@@ -49,13 +47,8 @@ export function AnnotationQueryEditor(props: Props) {
               aria-label="Set lower limit for the step parameter"
               placeholder={'auto'}
               minWidth={10}
-              onCommitChange={(ev) => {
-                onAnnotationChange({
-                  ...annotation,
-                  step: ev.currentTarget.value,
-                });
-              }}
-              defaultValue={query.interval}
+              value={query.interval}
```

Comment:
I see a warning on the console when I change the value.

`An AutoSizeInput is changing from an uncontrolled to a controlled input. If you want to control the input, the empty value should be an empty string. Error Component Stack`